lib/AFS/CellCC/Dump.pm

```diff
@@ -207,9 +207,9 @@ _do_dump($$$) {
     my $dump_size = _get_size($job, $volname, $server, $partition, $lastupdate);
     DEBUG "got dump size $dump_size for volname $volname";
 
-    if (!scratch_ok($job, $prev_state, $dump_size,
+    if (!scratch_ok($prev_state, $dump_size,
                     config_get('dump/scratch-dir'),
-                    config_get('dump/scratch-minfree'))) {
+                    config_get('dump/scratch-minfree'), $job)) {
```

Comment:
a comment for the whole commit: these refactoring commits I think should be all done as one single commit. They all do the same thing, just for different functions, and keeping them separate leaves some code in this weird "in-between" state where some of these functions take arrays of jobs, and others take single jobs (and some of these are really small/mundane changes). This line, for instance, calls scratch_ok with a single job, but we're never going to actually see that line stay like that, since in the future it's going to change to pass in an array of jobs; the same array that was given to us.